### PR TITLE
Feat/condo/doma-7445/invoice context model

### PR DESCRIPTION
**Pull request by AleX83Xpert**
This PR includes the `InvoiceContext` model. This model will be used for marketplace settings for a particular organization.
Later, we may add the VAT value and the tax regime.

### CODE REVIEW

File: apps/condo/domains/marketplace/schema/InvoiceContext.test.js [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+                })
+            })
+        })
+
+        describe('hard delete', () => {
+            test('admin can\'t', async () => {
+                const [o10n] = await createTestOrganization(adminClient)
+                const [objCreated] = await createTestInvoiceContext(adminClient, o10n)
+
+                await expectToThrowAccessDeniedErrorToObj(async () => {
```
Comment (VKislov) [inline apps/condo/domains/marketplace/schema/InvoiceContext.test.js:159]: Accessories are checked before searching the database, so you don’t need to create entities to check for deletion; you can try to delete a non-existent record

File: apps/condo/domains/marketplace/schema/InvoiceContext.test.js [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+                expect(objs1).toEqual([expect.objectContaining({ id: objCreated1.id })])
+                expect(objs2).toEqual([expect.objectContaining({ id: objCreated2.id })])
+            })
+
+            test('anonymous can\'t', async () => {
+                const [o10n] = await createTestOrganization(adminClient)
+                await createTestInvoiceContext(adminClient, o10n)
+
+                const client = await makeClient()
+                await expectToThrowAuthenticationErrorToObjects(async () => {
```
Comment (VKislov) [inline apps/condo/domains/marketplace/schema/InvoiceContext.test.js:273]: Accessories are checked before searching the database, so you don’t need to create entities to check for anonymus

**Review by VKislov — APPROVED** [review]

File: apps/condo/domains/marketplace/schema/InvoiceContext.test.js (hunk 2 above)
Comment (AleX83Xpert) [reply apps/condo/domains/marketplace/schema/InvoiceContext.test.js:270]: fixed

**Review by VKislov — APPROVED** [review]

File: apps/condo/domains/marketplace/schema/InvoiceContext.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+        status: {
+            schemaDoc: 'The current status of the context',
+            isRequired: false,
+            type: 'Select',
+            dataType: 'string',
+            options: INVOICE_CONTEXT_STATUSES,
+        },
+
+        implicitFeePercent: {
```
Comment (pahaz) [inline apps/condo/domains/marketplace/schema/InvoiceContext.js:79]: Need isAdmin access?

Comment (AleX83Xpert) [reply apps/condo/domains/marketplace/schema/InvoiceContext.js:79]: It's improved in the next PR
https://github.com/open-condo-software/condo/pull/3996/files#diff-b5fe9d504df9cf856d6dca749c674a530c463b3bfa3d5d512081aed05d81b4a6R101

**Pull request merged**